ngraph/frontend/paddlepaddle/src/op/elementwise_ops.cpp

```diff
@@ -0,0 +1,102 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <map>
+
+#include <ngraph/opsets/opset6.hpp>
+#include "elementwise_ops.hpp"
+
+namespace ngraph
+{
+    namespace frontend
+    {
+        namespace pdpd
+        {
+            namespace op
+            {
+                template <typename T>
+                NamedOutputs elementwise_ops(const NodeContext& node)
+                {
+                    auto x = node.get_ng_input("X");
+                    auto y = node.get_ng_input("Y");
+
+                    auto axis = node.get_attribute<int>("axis");
+
+                    PDPD_OP_VALIDATION_CHECK(node,
+                                             x.get_partial_shape().rank().is_static(),
+                                             "elementwise_ops: X rank must be static!");
+                    PDPD_OP_VALIDATION_CHECK(node,
+                                             y.get_partial_shape().rank().is_static(),
+                                             "elementwise_ops: Y rank must be static!");
```

Comment:
Ticket 58539 is created for China team to handle this

Reply:
@liubo-intel Please have an investigation

Reply:
@jane-intel There will only be static rank to inputs.  Confirmed with Baidu PaddleLite arch and Paddle op engineers. So let's defer this rank relax implementation. It will also make the converted nGraph function simple and friend to performance.